src/styles/main.scss

```
@@ -0,0 +1,54 @@
+@import 'variables';
+@import 'product-card';
+@import 'header';
+
+.showcase {
+  --column-count: 4;
+
+  display: grid;
+
+  grid-template-columns: repeat(var(--column-count), 200px);
+  padding: 50px 40px;
+  justify-content: center;
+  gap: 48px 46px;
```

try to switch this params to pass tests.
```suggestion
   gap: 46px 48px;
```